pkg/server/conn.go

```diff
@@ -1106,7 +1106,7 @@ func (cc *clientConn) Run(ctx context.Context) {
 				server_metrics.DisconnectNormal.Inc()
 				return
 			} else if terror.ErrResultUndetermined.Equal(err) {
-				logutil.Logger(ctx).Error("result undetermined, close this connection", zap.Error(err))
```

Comment:
This should be an ERROR?
Transaction result undetermined, so we force close the connection to tell the pass this information to the client.